service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsController.java

```diff
@@ -49,25 +57,32 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
     public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody CallbackRequest callbackrequest) {
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
 
+        // !! IF NEEDED !!
+        // Maybe add the storage preparation here or in the submit-event
```

Comment:
what's this for?

Reply:
Don't exactly know how ccd store the dynamic list yet. Do the prepare it and remove all the other values apart from the selected one or do they not. If they don't then we may want to do it manually to save on storage.

Reply:
Seeing as we are not using the dynamic list anymore then we don't need to worry about it